managed-databases/postgresql-and-mysql/api-cli/verify-ca-postgresql.mdx

```diff
@@ -0,0 +1,256 @@
+---
+meta:
+  title: Verifying Servers' Certificate Authority on PostgreSQL
+  description: This page explains how to verify a server's certificate authority on Managed Database for PostgreSQL.
+content:
+  h1: Verifying Servers' Certificate Authority on PostgreSQL
+  paragraph: This page explains how to verify a server's certificate authority on Managed Database for PostgreSQL.
+tags: verify-ca verify certificate authority postgresql
+dates:
+  validation: 2023-04-01
+  posted: 2023-04-01
+categories:
+  - managed-databases
+---
+
+The creation and management of TLS certificates is included in the Managed Database for PostgreSQL offers.
+
+A TLS certificate is generated automatically for your Database Instance upon its creation.
+
+You can renew your certificates at any time, when you add a new endpoint, for example.
+
+PostgreSQL natively supports SSL connections that enhance the security of client/server communications with TLS protocols. PostgreSQL does not, however, verify server certificates by default.
+
+In this page we show you two types of protection modes you can use to verify if the servers connecting to your client are properly encrypted.
```

Comment:
```suggestion
In this page we show you two types of protection modes you can use to verify if the servers connecting to your client are properly encrypted.
```
In this page, we show you two types of protection modes you can use to verify whether the servers connecting to your client are properly encrypted.